crates/metastore/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = {workspace = true}
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
Minor nitpick, but, we would want to add errors to the enum instead of using `anyhow` in this crate.